File: CodeForces/game_with_string.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){
    string word;
    cin>>word;
    stack<int> st;
    st.push(word[0]);
    int count=0;
    for(int a=1;a<word.length();a++){
        if(word.length()>0 && st.top()==word[a])
        count++;
        else
        st.push(word[a]);
    }
    if(count%2==1){
        cout<<"Yes";
    }
    else cout<<"No";
}

File: CodeForces/game_with_string.py
from collections import deque
word=input()
st=deque()
st.append(word[0])
count=1
for i in word[1::]:
    if len(st)>0 and i==st[len(st)-1]:
        st.pop()
        count+=1
    else:
        st.append(i)
if count%2==1:
    print('No')
else:
    print('Yes')